components/theme.ts

```diff
@@ -10,6 +10,17 @@ export const theme = extendTheme({
     lightGray: '#F1F1F1',
     black: '#000000',
     white: '#FFFFFF',
+    Primary: {
+      50: '#E3FCF5',
+      500: '#20A1A8',
+      600: '#178090',
+      700: '#106178',
+      800: '#0A4761',
+    },
+    Gray: {
+      500: '#667085',
+      900: '#101828',
+    },
```

Comment:
Nested colors are referenced by dot notation - [see docs](https://v1.chakra-ui.com/docs/styled-system/features/style-props#color-and-background-color).